### PR TITLE
Add feature to support centered anchoring.

### PR DESCRIPTION
**Pull request by fabriciopolonio**
I have built all my adapters for the copper-clad boards to fit the center of the LCD display of my Photon M3, I assume other people might as well have the same.

It would be great if this could be incorporated into the main repository.

image
image

### CODE REVIEW

File: src/renderer/render-to-photon.js [hunk 1]
```diff
@@ -128,11 +128,15 @@ const {CanvasToImage} = require("./canvas_to_img");
...
+            "BL": [x_offset_px, outputHeight - y_offset_px - board_height_px],
             "BR": [
-                longRes - x_offset_px - board_width_px,
-                shortRes - y_offset_px - board_height_px
+                outputWidth - x_offset_px - board_width_px,
+                outputHeight - y_offset_px - board_height_px
+            ],
+            "C": [
+                (outputWidth / 2) - x_offset_px - (board_width_px / 2),
+                (outputHeight / 2) - y_offset_px - (board_height_px / 2)
```
Comment (Andrew-Dickinson) [inline src/renderer/render-to-photon.js]:
```suggestion
                (outputWidth / 2) + x_offset_px - (board_width_px / 2),
                (outputHeight / 2) + y_offset_px - (board_height_px / 2)
```

**Review by Andrew-Dickinson — CHANGES_REQUESTED** [review]
This looks great @fabriciopolonio. Thanks for opening the PR!

I like the feature, though it interacts a bit poorly with the current requirement that "Anchor Offset" values be non-negative. Upon further reflection, I don't think we need to babysit our users quite that much. If they ask us to render their board starting -20mm off of the exported viewport, they probably have a good reason for it, or will notice that a chunk of their board is missing. I've changed this behavior in `1.2.0` to allow negative inputs, and suggested a change below which makes the sign convention a bit nicer for the center anchor.

Could you rebase to 8cfa8122b5a6a86d0254da5499b32246946733d5, incorporate the below suggestion, and resubmit with a version bump in `package.json` taking us to `1.3.0`?

File: src/renderer/render-to-photon.js (hunk 1 above)
Comment (fabriciopolonio) [reply src/renderer/render-to-photon.js]: Hi @Andrew-Dickinson, sorry the late answer.

I have performed the changes you have requested, let me know if anything else is needed.

**Comment by Andrew-Dickinson** [comment]
Looks great, merged! Thanks again @fabriciopolonio

**Pull request merged**